python/hsfs/core/online_store_sql_client.py

```diff
@@ -42,11 +42,12 @@ def __init__(
         feature_store_id: id,
         skip_fg_ids: Optional[Set[int]],
         serving_keys: Optional[Set[ServingKey]] = None,
+        external: bool = True,
     ):
-        _logger.info("Initialising OnlineStoreSqlClient")
+        _logger.info("Initialising Online Store Sql Client")
```

Comment:
not strictly necessary, may be also here change the log level to debug?

Reply:
We can although I think it is good to have as otherwise in a notebook one wonders what is happening